release/package.json

```diff
@@ -155,43 +155,53 @@
       },
       {
         "command": "MSBuild.buildCurrent",
-        "title": "MSBuild: Build current project"
+        "title": "MSBuild: Build current project",
+        "when": "fsharp.debuggerAvailable"
```

Comment:
YAK: should we nest `fsharp.debugger.available` or keep it flat?

Reply:
nested works for me - I'll do the global rename once I get all of the correct commands hooked up to the key